entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md

```diff
@@ -1711,3 +1712,38 @@ modelBuilder
      .HasOne(e => e.UpdatedBy)
      .WithMany();
 ```
+
+<a name="udf-empty-string"></a>
+### DbFunction.Schema being null or empty string configures it to be in model's default schema
+
+[Tracking Issue #12757](https://github.com/aspnet/EntityFrameworkCore/issues/12757)
+
+This change is introduced in EF Core 3.0-preview 7.
+
+**Old behavior**
+
+A DbFunction with value of schema being empty string configured the function to be in no schema and treated it as built-in function. For example following code will map `DatePart` CLR function to `DATEPART` built-in function on SqlServer.
+
+```C#
+[DbFunction("DATEPART", Schema = "")]
+public static int? DatePart(string datePartArg, DateTime? date) => throw new Exception();
+
+```
+
+**New behavior**
+
+All DbFunction mappings are considered to be mapped to user defined functions. Hence empty string value would put the function inside default schema for the model. It can be default schema configured explicitly via fluent API `modelBuilder.HasDefaultSchema()` or `dbo`.
```

Comment:
```suggestion
All DbFunction mappings are considered to be mapped to user defined functions. Hence empty string value would put the function inside the default schema for the model. Which could be the schema configured explicitly via fluent API `modelBuilder.HasDefaultSchema()` or `dbo` otherwise.
```